tests/js/spec/views/eventsV2/saveQueryButton.spec.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import {browserHistory} from 'react-router';
+
+import EventSaveQueryButton from 'app/views/eventsV2/saveQueryButton';
+import {ALL_VIEWS} from 'app/views/eventsV2/data';
+import EventView from 'app/views/eventsV2/eventView';
+
+describe('EventsV2 > SaveQueryButton', function() {
+  const errorsView = EventView.fromEventViewv1(
+    ALL_VIEWS.find(view => view.name === 'Errors')
+  );
+  let organization, location;
+  beforeEach(function() {
+    organization = TestStubs.Organization();
+    location = {
+      pathname: '/organization/eventsv2/',
+      query: {},
+    };
+  });
+
+  it('renders a button', function() {
+    const wrapper = mount(
+      <EventSaveQueryButton
+        organization={organization}
+        location={location}
+        isEditing={false}
+        eventView={errorsView}
+      />,
+      TestStubs.routerContext()
+    );
+    const button = wrapper.find('StyledDropdownButton');
```

Comment:
When we query for buttons, can we search via `aria-label` - this way it helps make sure our buttons are accessible.